basic_autonomy_ros2/test/test_waypoint_generation.cpp

```diff
@@ -778,12 +778,61 @@ TEST(BasicAutonomyTest, get_nearest_basic_point_index)
         std::vector<basic_autonomy::waypoint_generation::PointSpeedPair> points = basic_autonomy::waypoint_generation::create_lanefollow_geometry(maneuver, 
                                                                                     starting_downtrack, wm, general_config, detailed_config, visited_lanelets);
 
-        EXPECT_EQ(visited_lanelets.size(), 5);
+        EXPECT_EQ(visited_lanelets.size(), 3);
```

Comment:
Why is this test returning 3 but the ros1 test returned 2?
Seems like there might be a mismatch in the implementations.

Reply:
Both should be 3. In ros1 I checked the test before adding the extra buffer lanelet and forgot to update